components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -0,0 +1,106 @@
+<chef-page
+  subheading="Resources that match the type and all conditions are included in the project."
+  confirm-btn-text="Save Rule"
+  [attr.heading]="getHeading()"
+  [attr.disable-confirm]="!ruleForm.valid"
+  [attr.page-loading]="isLoading"
+  [attr.confirm-loading]="saving"
+  (confirm)="saveRule()"
+  (close)="closePage()">
+  <div *ngIf="!isLoading">
+    <ng-container>
+      <form id="ruleForm" [formGroup]="ruleForm" (ngSubmit)="saveRule()">
+
+        <chef-form-field class="display3">
+          <label>
+            <span class="label">Rule Name</span>
+            <chef-input id="ruleName" ngDefaultControl formControlName="name"></chef-input>
+          </label>
+          <chef-error
+            *ngIf="ruleForm.get('name').hasError('required') && ruleForm.get('name').dirty"
+          >Name is required</chef-error>
+        </chef-form-field>
+
+        <chef-form-field class="display3">
+          <label>
+            <span class="label">Resource Type</span>
+            <select id="resourceType" ngDefaultControl formControlName="type">
+              <option value='NODE'>Node</option>
+              <option value='EVENT'>Event</option>
+            </select>
+          </label>
+          <chef-error
+            *ngIf="ruleForm.get('type').hasError('required') && ruleForm.get('type').dirty"
+          >Resource Type is required</chef-error>
+          <small>Resource types cannot be changed after the rule is created.</small>
+        </chef-form-field>
+
+        <div id="add-condition">
+          <div class="label">Conditions</div>
+          <small>All conditions must be true for the rule to be true.</small>
+          <chef-button primary (click)="addCondition()">Add Condition</chef-button>
+        </div>
+
+        <div id="condition-list" formArrayName="conditions">
+
+          <div class="condition-item"
+            [formGroupName]="i"
+            *ngFor="let condition of ruleForm.get('conditions').controls; let i = index">
+
+            <chef-form-field class="attribute-field">
+              <label>
+                <span class="label">{{ ruleForm.get('type').value | uppercase }} Attribute</span>
+                <select ngDefaultControl formControlName="attribute">
+                  <option value='CHEF_SERVER'>Chef Server</option>
+                  <option value='CHEF_ORGS'>Chef Organization</option>
```

Comment:
This list should not be hard-coded. The items in the list need to update dynamically based on the resource type selection, as itemized on https://chefio.atlassian.net/browse/A2-318.

Reply:
oh sorry this is my fault, i told @tarablack01 this was okay and the auth team could do that update. would you prefer that get addressed in this pr @msorens?

Reply:
No preference--I am just going by the spec. 😁

Reply:
here are the allowed attributes for each rule type ([from this story](https://chefio.atlassian.net/browse/A2-318)):
```
NODE ATTRIBUTES
Chef Organization (string)
Chef Server (string)
Environment (string)
Chef Role (string[])
Chef Tag (string[])
Chef Policy Name (string)
Chef Policy Group (string)

EVENT ATTRIBUTES
Chef Organization (string)
Chef Server (string)
```